README.md

```diff
@@ -0,0 +1,36 @@
+# netzstrategen Gulp tasks collection
+
+Commonly used Gulp tasks shared across projects.
+
+## Supported tasks
+
+- Clean `/dist` folder
+- Sass compilation
+- JS compilation
+- Image compression
+- Fractal server and build
+
+## Installation
+
+1. `yarn add @netzstrategen/gulp-task-collection`
+2. `require` in project Gulpfile (see [sample.gulpfile.js](https://github.com/netzstrategen/gulp-task-collection/blob/master/sample.gulpfile.js))
+3. Customise asset paths in project `package.json` (see [package.json](https://github.com/netzstrategen/gulp-task-collection/blob/master/package.json))
+
+## Available commands
+
+- `gulp`: Default task runs (in order) `fractal:start`, `images`, `styles`, `scripts` and then watches for changes
+- `gulp build`: Build assets (`images`, `styles`, `scripts`)
+- `gulp build:production`: Build production-ready assets
+
+
+- `gulp clean`: Clean `/dist` folder
+- `gulp styles` and `gulp styles:production`: Sass compilation
```

Comment:
Should we clarify the difference between `build` and `build:production`?

Reply:
Is it not clear in the task name? If one needs more detailed information then they can take a look at the task file.

Reply:
Ok, I also think, people who work with these are familiar with this terminology.